### PR TITLE
Add list of desired automation tasks

### PR DESCRIPTION
**Pull request by avanti-joshi**
Changes proposed in this pull request:

- Add initial list of automated tasks brainstorms

### CODE REVIEW

File: docs/how-we-work/automated-actions.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# List of processes we would like to automate 
+
+## Goals and Action Items:
+
+### Make Jenkins more secure and use to decrease technical toil 
+* Move Jenkins behind the VPN: [OHSH-356](https://ocio-jira.acf.hhs.gov/browse/OHSH-356)
+* Add additional Jenkins workers to execute jobs: [OHSH-778](https://ocio-jira.acf.hhs.gov/browse/OHSH-378)
+* Document our standard for where we configure Jenkins jobs
```
Comment (ssatagopan) [inline docs/how-we-work/automated-actions.md:8]: Jenkins job to update staging and dev ahead of production maintenance: [OHSH-355](https://ocio-jira.acf.hhs.gov/browse/OHSH-355)
Jenkins script to restart MariaDB: [OHSH-85](https://ocio-jira.acf.hhs.gov/browse/OHSH-85)

**Review by ssatagopan — CHANGES_REQUESTED** [review]
Added a couple of existing tickets, otherwise LGTM

File: docs/how-we-work/automated-actions.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* Create Jenkins jobs to run necessary yum commands as part of maintenance
+* Automate testing in between above jobs as part of maintenance testing
+* Write Jenkins job (using Ansible scripts) to switch varnish1 to point at lifeboat etc etc so we dont’ need to support drupal deployments
+
```
Comment (kilbergr) [inline docs/how-we-work/automated-actions.md:12]: can we also evakuate and add/rm Jenkins access for team members as is reasonable (audit)?

File: docs/how-we-work/automated-actions.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+### Make it easier to get instances to a known configuration
+* Documentation on how to use Ansible
+* Trigger specified updates on specific hosts
```
Comment (kilbergr) [inline docs/how-we-work/automated-actions.md:15]: Like pkg updates?

Comment (kilbergr) [reply docs/how-we-work/automated-actions.md:15]:
```suggestion
* Trigger specified updates on specific hosts
* Trigger specified package removal on specific hosts
```

File: docs/how-we-work/automated-actions.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* Create user accounts/remove user accounts - document the process of removing user accounts. Audit which users exist on all the systems
+* Create sudo access
+* Management of user access on VPNs (east and west)
+* Investigate automating logging configuration on hosts
```
Comment (kilbergr) [inline docs/how-we-work/automated-actions.md:19]:
```suggestion
* Investigate automating logging configuration on hosts (including logrotate)
```

File: docs/how-we-work/automated-actions.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+### Make it so lifeboat and maintenance pages are easier to manage
+* Write ansible jobs to keep Lifeboat up to date.
+* Jenkins job to enable/disable maintenance pages (likely also leveraging ansible): [OSH-215](https://ocio-jira.acf.hhs.gov/browse/OHSH-215)
+
+### Make it so we can make West match East
+* Take AMI snapshots of East nightly + sync to West. (with cleanup) [OHSH-239](https://ocio-jira.acf.hhs.gov/browse/OHSH-239)
+* Create a job to reconstitute West from ami snapshots
+
+### Maintenance prep is automated
+* Automatically run latest update on Stage and output report
```
Comment (kilbergr) [inline docs/how-we-work/automated-actions.md:31]: Could we have a reach goal to automate creation of JIRA tickets when security vulns are found in scans (Qualys etc)?

File: docs/how-we-work/automated-actions.md (hunk 2 above)
Comment (kilbergr) [inline docs/how-we-work/automated-actions.md:12]:
```suggestion
* [Create a jenkins job to restart MariaDB server](https://ocio-jira.acf.hhs.gov/browse/OHSH-85)
```

**Review by kilbergr — COMMENTED** [review]
Lots of great IDEAS! Added some clarifying qs and a couplr of additions. Thank you !

**Review by kilbergr — APPROVED** [review]
lgtm, thank you!

**Review by ssatagopan — APPROVED** [review]

**Pull request merged**